scipy/optimize/_linprog.py

```diff
@@ -557,10 +568,21 @@ def linprog(c, A_ub=None, b_ub=None, A_eq=None, b_eq=None,
     # need modified bounds here to translate variables appropriately
     disp = solver_options.get('disp', False)
 
-    x, fun, slack, con, status, message = _postprocess(x, postsolve_args,
-                                                       complete, status,
-                                                       message, tol,
-                                                       iteration, disp)
+    # print("linprog() bounds: input to postsolve ", postsolve_args[0][5])
+
+    # x, fun, slack, con, status, message = _postprocess(x, postsolve_args,
+    #                                                    complete, status,
+    #                                                    message, tol,
+    #                                                    iteration, disp)
+
+    x, fun, slack, con = _postsolve(x, postsolve_args, complete, tol)
```

Comment:
`tol` is no longer an argument of `_postsolve`, though. Looks like it's going in as `copy`. But it looks like `copy` is not used anymore, either.

Reply:
The `tol` argument should be removed here.
I don;t know about `copy`: it is only used once (set to `True`) in `_linprog_rs.py`, `_phase_two()`. I haven't dug into what this exactly does. Do you know?

Reply:
I don't remember. I'll take another look when you'd like me to review again.

Reply:
I'll remove the parameter completely.
I went through your comments and made the necessary changes. Do you prefer me to push the results before you continue reviewing?

Reply:
Please go ahead and push and I'll review.